packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -157,6 +157,8 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
      * Renders the component
      */
     public render(): React.ReactElement<HTMLDivElement> {
+        // we set role="combobox" in order to get assistive technologies to enumerurate
```

Comment:
nit: spelling "enumerate", "focused", also I think a little copy editing might make this more understandable, I am still not quite sure what setting the role to combobox is achieving from this comment. What does it prevent from happening, what does it ensure happens? I just see a list of properties. Does it read them in that order? If so, I would also reword that to be more like "to enumerate a focused trigger elements label, value, role and whether the listbox is expanded".

Reply:
Change to a more concise  
// we set role="combobox" in order to match how native select elements
// are described by assistive technologies